example.py

```diff
@@ -152,7 +152,10 @@ def parse_cmdline():
     parser = ArgumentParser(description=__doc__)
     parser.add_argument(
         "--dsn",
-        default="postgresql://maxroach@localhost:26257/bank?sslmode=disable",
+        # For cockroach demo:
+        default="postgresql://<username>:<password>@<hostname>:<port>/bank?sslmode=require",
```

Comment:
It is not a valid default. `cockroach demo` changes the password and the port every time you boot up a virtual cluster, so you have to manually change these parameters. We outline this in the tutorial at https://www.cockroachlabs.com/docs/stable/build-a-python-app-with-cockroachdb.html.

Reply:
I understand, but it seems more appropriate to have it in the help description, or have it as the argument `metavar`, than as the argument default.

Being the default, that value will be used if the parameter is not specified, and will make connecting to fail with some random error message, rather than failing with a message such as "please specify a connection string"

Reply:
I understand your point, @dvarrazzo. I'm waiting to see if it's acceptable for us to use a sample username and password in this example, from a security standpoint. If so, I can update this. I think I'll have to use a default host and port and ask the user to replace it, given what Eric says about `cockroach demo`. I'll update this soon once we've nailed this down.

Reply:
I agree that this should be functional code but would like to avoid providing a default password since we know that `cockroach demo` won't have a consistent password anyway. Fwiw I'd be in favor of an arg with a `metavar` as @dvarrazzo mentioned and emit an error if the user has not collected and specified the password to the app.

Reply:
The latest commit includes a couple changes:

- Changed `dsn` to be a required, positional argument. If it's not provided, an error is returned.
- Moved over dsn value suggestions to the help text for the `dsn` argument.
- Removed the default value for dsn, as no default value will work out-of-the-box.

I didn't add a `metavar`, since the demo and CC strings use different SSL options.